packages/wdio-browserstack-service/src/config.ts

```diff
@@ -0,0 +1,47 @@
+import type { AppConfig, BrowserstackConfig } from './types.js'
+import type { Options } from '@wdio/types'
+import TestOpsConfig from './testOps/testOpsConfig.js'
+import { isUndefined } from './util.js'
+
+class BrowserStackConfig {
+    static getInstance(options?: BrowserstackConfig & Options.Testrunner, config?: Options.Testrunner): BrowserStackConfig {
+        if (!this._instance && options && config) {
+            this._instance = new BrowserStackConfig(options, config)
+        }
+        return this._instance
+    }
+
+    public userName?: string
+    public accessKey?: string
+    public framework?: string
+    public buildName?: string
+    public buildIdentifier?: string
+    public testObservability: TestOpsConfig
+    public percy: boolean
+    public accessibility: boolean
+    public app?: string|AppConfig
+    private static _instance: BrowserStackConfig
+    public appAutomate: boolean
+    public automate: boolean
+    public funnelDataSent: boolean = false
+
+    constructor(options: BrowserstackConfig & Options.Testrunner, config: Options.Testrunner) {
```

Comment:
Seems like this is suppose to be a singleton
```suggestion
    private constructor(options: BrowserstackConfig & Options.Testrunner, config: Options.Testrunner) {
```